composer.json

```diff
@@ -43,20 +43,22 @@
         "symfony/mailer": "4.4.*",
         "symfony/maker-bundle": "^1.19",
         "symfony/monolog-bundle": "^3.5",
-        "symfony/profiler-pack": "^1.0",
         "symfony/security-bundle": "4.4.*",
         "symfony/swiftmailer-bundle": "^3.4",
         "symfony/translation": "4.4.*",
-        "symfony/twig-pack": "^1.0",
```

Comment:
Again, why was this updated? Can this be a composer 2 doing?

Reply:
This is the same reason as previously mentioned.
see: https://packagist.org/packages/symfony/twig-pack

Reply:
Due to a composer require all the other packages are updated too. The update will bring no other changes next to the symfony/twig-pack getting deleted. This is because the other packages are part of the pack and are also installed.